java/test/pom.xml

```diff
@@ -117,6 +117,41 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
```

Comment:
Shouldn't we also update `java/api/pom.xml` and `java/runtime/pom.xml`?

Reply:
Not necessary, those pom files extends from super pom

Reply:
But `java/test/pom.xml` does not?

Reply:
All java sub module extends from `java/pom.xml`